Documentation/network/servicemesh/ingress.rst

```diff
@@ -43,6 +43,8 @@ path prefix) between resources.
 This is a step-by-step guide on how to enable the Ingress Controller in
 an existing K8s cluster with Cilium installed.
 
+.. include:: ingress-prerequisites.rst
+
```

Comment:
These prerequisites are unlikely to ever be included in another page, so I'd just drop them in ingress.rst instead of doing a separate ingress-prerequisites.rst.

Same for l7-traffic-management.rst.